rules/microsoft_rules/microsoft_exchange_external_forwarding.py

```diff
@@ -0,0 +1,30 @@
+ALLOWED_DOMAINS = ["company.com"]
+
+EMAIL_EXCEPTIONS = ["exception@example.com"]
```

Comment:
if I'm reading this detection correctly, `EMAIL_EXCEPTIONS` means  `ALLOWED_FORWARDING_DESTINATION_EMAILS` and `ALLOWED_DOMAINS` means `ALLOWED_FORWARDING_DESTINATION_DOMAINS`

Reply:
yup i'll update the names to be more clear